modules/ocm-overview-tab.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// ocm/ocm-overview.adoc
+
+[id="ocm-overview-tab_{context}"]
+= Overview tab
+
+The overview tab provides information about how your cluster was configured.
+
+1. **Cluster ID** is the unique identification for the created cluster. This ID can be used when issuing commands to the cluster via the command line.
+1. **Type** shows the OpenShift version that the cluster is using.
+1. **Region** is the server region.
+1. **Provider** shows which cloud provider that the cluster was built upon.
+1. **Availability** shows which type of availability zone that the cluster uses, either single or multizone.
+1. **Version** is the OpenShift version that is installed on the cluster. If there is an update available, you can update from this field.
+1. **Created at** shows the date and time that the cluster was created.
+1. **Owner** identifies who created the cluster and has owner rights.
+1. **Subscription type** shows the subscription model that was selected on creation.
+1. **Infrastructure type** is the type of account that the cluster uses.
+1. **Status** displays the current status of the cluster.
+1. **Total vCPU** shows the total available virtual CPU for this cluster.
+1. **Total memory** shows the total available memory for this cluster.
+1. **Load balancers**
```

Comment:
No introduction for `Load balancers` ?

Reply:
@xueli181114 and @gregsheremeta I really didn't know what would go here? Any ideas?